### ISSUE TITLE
Buffer is not defined, theme_apiDoc error on latest docusuarus version

### ISSUE THREAD

**Issue by sibelius** [rohit-gohri/redocusaurus#37]
config

```jsx
module.exports = {
[
      'redocusaurus',
      {
        specs: [
          {
            routePath: '/api-public/',
            spec: './src/swaggers/public.json',
          },
          {
            routePath: '/api-private/',
            spec: './src/swaggers/public.json',
          },
        ],
      },
    ],
}

error

```jsx
Docusaurus Node/SSR could not render static page with path=/api-public because of error: Buffer is not defined
Docusaurus Node/SSR could not render static page with path=/api-private because of error: Cannot access 'theme_ApiDoc' before initialization

ReferenceError: Buffer is not defined
ReferenceError: Cannot access 'theme_ApiDoc' before initialization
at Array.forEach (<anonymous>)
  error building locale=pt-BR
  Error: Failed to compile with errors.
  at /app/node_modules/@docusaurus/core/lib/webpack/utils.js:198:24
  at /app/node_modules/webpack/lib/MultiCompiler.js:550:14
  at processQueue (/app/node_modules/webpack/lib/MultiCompiler.js:488:6)
  at processTicksAndRejections (internal/process/task_queues.js:75:11)
```

version
```jsx
"@docusaurus/core": "2.0.0-beta.0",
"@docusaurus/preset-classic": "2.0.0-beta.0",
"redocusaurus": "^0.3.4",
```

**Comment by rohit-gohri** [comment]
Related to #25 I think. Testing with latest beta currently

**Comment by rohit-gohri** [comment]
Fixed in v0.4.0
